ui-pf4/src/main/webapp/src/pages/projects/new-project/review/review.tsx

```diff
@@ -192,12 +204,19 @@ export const Review: React.FC<ReviewProps> = ({ match, history: { push } }) => {
                       .join(", ")}
                   </DescriptionListDescription>
                 </DescriptionListGroup>
-                {/* <DescriptionListGroup>
-                  <DescriptionListTerm>Annotation</DescriptionListTerm>
+                <DescriptionListGroup>
```

Comment:
Do you think Source would be better to appear before than Target ? seems more natural to show Source,Target ....

Reply:
@jonathanvila I know what you mean and I agree, it is more natural to have Source and then Target; however, in the advanced options we are showing first Target and then Source (see image below), the order was set based on priority I think. I would show the same order as the "Advanced options" but I'm open to suggestions.

![Screenshot from 2020-11-13 12-19-39](https://user-images.githubusercontent.com/2582866/99067310-c8e43880-25aa-11eb-9afe-602a08484c45.png)

Reply:
Sure, if the "Advaced options" has a reason to appear in that order seems right to follow same criteria in the summary. :+1: